docs/static/_redirects

```diff
@@ -13,6 +13,10 @@ https://next.material-ui.com/* https://material-ui.com/:splat 301!
 https://material-ui-next.com/* https://material-ui.com/:splat 301!
 https://material-ui.dev/* https://material-ui.com/:splat 301!
 
+# old code links
+# likely /style/typography#migration-to-typography-v2
+/style/typography https://v3-9-0.material-ui.com/style/typography/#migration-to-typography-v2 301
```

Comment:
This creates an infinite SEO redirection loop. I don't know the implications.

Reply:
```suggestion
https://material-ui.com/style/typography https://v3-9-0.material-ui.com/style/typography/#migration-to-typography-v2 301
```
?

Reply:
The redirection loop comes from the canonical link of the v3-9-0 documentation page. If I had to guess, Google will unindex the page when he sees a redirection. It's probably OK.

Reply:
You mean 
> /style/* /components/:splat 301/style/* /components/:splat 301

Should we change those since they're incorrect for older documentation versions?

Reply:
view-source:https://v3-9-0.material-ui.com/style/typography/

```html
…<link rel="canonical" href="https://material-ui.com/style/typography/"/><link r…
```

Reply:
> Should we change those since they're incorrect for older documentation versions?

Older versions of the documentation will be using older versions of the redirects, since each is built from a separate branch.

Reply:
@mbrookes Right. I was trying to guess what olivier means. Canonical discussion is continued at https://github.com/mui-org/material-ui/pull/14806#discussion_r291090134